src/Client.php

```diff
@@ -405,6 +405,11 @@ private function applyOptions(RequestInterface $request, array &$options): Reque
                     $options['curl'][CURLOPT_HTTPAUTH] = CURLAUTH_NTLM;
                     $options['curl'][CURLOPT_USERPWD] = "$value[0]:$value[1]";
                     break;
+                case 'bearer':
+                    // Ensure that we don't have the Authorization header in
+                    // different case and set the new value wih the token
```

Comment:
```suggestion
                    // different case and set the new value with the token
```
Noticed this minor typo